src/pages/mesh/advanced/headers.md

```diff
@@ -18,6 +18,10 @@ To specify headers for your mesh, you can add them inside the `JSON` file that d
 
 Request headers provide more information about the request context. Currently, you can add request headers to your [mesh config](#add-request-headers-in-your-mesh-file).
 
+<InlineAlert variant="info" slots="text"/>
```

Comment:
Consider re-writing line 19. "Currently" seems like an artifact of a feature that has changed scope over time.